tests/stabtest/makefile

```diff
@@ -18,7 +18,7 @@ endif
 default: stabtest$(EXT)
 
 stabtest$(EXT): stabtest.cpp ../../readerwriterqueue.h ../../atomicops.h ../common/simplethread.h ../common/simplethread.cpp makefile
-	g++ $(PLATFORM_OPTS) -std=c++11 -Wpedantic -Wall -DNDEBUG -O3 stabtest.cpp ../common/simplethread.cpp -o stabtest$(EXT) -pthread $(PLATFORM_LD_OPTS)
+	g++ $(PLATFORM_OPTS) -std=c++11 -Werror -Wsign-conversion -Wpedantic -Wall -DNDEBUG -O3 stabtest.cpp ../common/simplethread.cpp -o stabtest$(EXT) -pthread $(PLATFORM_LD_OPTS)
```

Comment:
I don't like putting `-Werror` in the makefiles since there's inevitably warnings on systems other than those that were used to test on. `-Wsign-conversion` is fine though.

Reply:
No problem. I will remove `-Werror`.